vm/src/obj/objbyteinner.rs

```diff
@@ -47,15 +49,15 @@ impl TryFromObject for PyByteInner {
 }
 
 #[derive(FromArgs)]
-pub struct ByteInnerNewOptions {
+pub(crate) struct ByteInnerNewOptions {
     #[pyarg(positional_only, optional = true)]
     val_option: OptionalArg<PyObjectRef>,
     #[pyarg(positional_or_keyword, optional = true)]
     encoding: OptionalArg<PyStringRef>,
 }
 
 impl ByteInnerNewOptions {
```

Comment:
If the struct you're implementing a function for is `pub(crate)`, then just a `pub fn` will be effectively a `pub(crate) fn`.